Kajabi/sage-lib
# Assistant Documentation Updates

## Description
Updates documentation and converts Assistant to Sage component.

### Screenshots
No visual changes with these adjustments.


## Test notes


### Steps for testing
1. Visit http://localhost:4000/pages/breakout/object/assistant
2. Verify UI and documentation updates.


## Related

- N/A
